frontend/src/metabase/nav/components/LicenseTokenMissingBanner/LicenseTokenMissingBanner.tsx

```diff
@@ -0,0 +1,36 @@
+import { t } from "ttag";
+
+import { Banner } from "metabase/components/Banner";
+import Link from "metabase/core/components/Link";
+import { adminLicense } from "metabase/lib/urls";
+import { Flex, Text } from "metabase/ui";
+
+import styles from "./LicenseTokenMissingBanner.module.css";
+
+export const LicenseTokenMissingBanner = ({
+  onClose,
+}: {
+  onClose: () => void;
+}) => {
+  return (
+    <Banner
+      iconColor="var(--mb-color-text-white)"
+      className={styles.BannerContainer}
+      bg="var(--mb-color-background-inverse)"
+      align="flex-start"
```

Comment:
You've already set the align in the css module for the `. BannerContainer` class.
```suggestion
```

Reply:
Yes! 😅  https://github.com/metabase/metabase/pull/59030/commits/18525a6ad8b8bed4e4fea0289baa30b3057d323e